### PR TITLE
Fix documentation backend config indentation

### PR DESCRIPTION
**Pull request by bartoszcisek**
#### What this PR does
Fixes documentation where storage config examples for azure and swift were not indented sufficiently.
#### Which issue(s) this PR fixes or relates to

No issue found for this problem.
#### Checklist

- [ ] Tests updated
- [x] Documentation added
- [x] `CHANGELOG.md` updated - the order of entries should be `[CHANGE]`, `[FEATURE]`, `[ENHANCEMENT]`, `[BUGFIX]`

### CODE REVIEW

**Review by osg-grafana — APPROVED** [review]
Unblocking with the understanding that an engineer needs to approve this.

**Review by pstibrany — APPROVED** [review]
Thank you!

File: CHANGELOG.md [hunk 1]
```diff
@@ -83,6 +83,7 @@ Querying with using `{__mimir_storage__="ephemeral"}` selector no longer works.
 ### Documentation
 
 * [ENHANCEMENT] Document migration from microservices to read-write deployment mode. #3951
+* [BUGFIX] Added indentation to azure an swift backend definition.
```
Comment (pracucci) [inline CHANGELOG.md:86]: @bartoszcisek Do you mind to add this PR number (`#4263`) at the end of the CHANGELOG entry, please? Thanks!

**Comment by bartoszcisek** [comment]
@pstibrany added PR fro that https://github.com/grafana/mimir/pull/4273

**Pull request merged**